OpenBCI_32bit_Library_Definitions.h

```diff
@@ -330,6 +330,9 @@
 /** Set sample rate */
 #define OPENBCI_SAMPLE_RATE_SET '~'
 
+// Insert marker into the stream
```

Comment:
please follow format for comments in this section:
`/** Insert marker into the stream */`